<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>crearContato</class>
 <widget class="QDialog" name="crearContato">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family:century gothic;
	font-size:24px;
}

QPushButton{
	background:#0069BF;
	border-radius:10px;
	color:white;
}

QPushButton:hover{
	background:#008EFF;
}

QLineEdit{
	border-radius:14px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Crear Contacto</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Nombre:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leNombre"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Apellido Paterno:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lePaterno"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Apellido Materno:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leMaterno"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Telefono:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leTelefono"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_5">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#0069BF;
color: white;</string>
     </property>
     <property name="text">
      <string>Agregar Contacto</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>crearContato</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>562</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>590</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
